data/json/mapgen/farm_dairy.json

```diff
@@ -495,9 +495,9 @@
         "................................................................................................",
         "................................................................................................",
         "...........................................................|-ll-|---|l--|--|--------|...........",
-        "...........................................................|[bb[|^^_|,,,|[[|-|&%~~~(|...........",
-        "...........................................................l~bb~|~~~|,,,|~~+*|~````(|...........",
-        "...........................................................l~~~~|~~~|,,,|~~|-|~```~_|...........",
+        "...........................................................|[bb[|^^_|...|[[|-|&%~~~(|...........",
+        "...........................................................l~bb~|~~~|...|~~+*|~````(|...........",
+        "...........................................................l~~~~|~~~|...|~~|-|~```~_|...........",
```

Comment:
These are meant to be open air terrain. Since I defined `"fill_ter": "t_open_air"`, there's no need in any change.

Reply:
I plan to make it so that the only legal characters for 'just `fill_ter`' are `.` and ` `, so this does need to change.  If you'd like to keep `,` as a visually distinct character I'll add it to the `terrain` list, but I feel like `.` works just as well here.

Reply:
Oh, I'm kinda misunderstand your intention. Ok, `.` is fine.